docs/source/models.rst

```diff
@@ -70,16 +70,10 @@ Migrating to the new API is very straightforward. The following method calls bet
     # Using pretrained weights:
     resnet50(weights=ResNet50_Weights.IMAGENET1K_V1)
     resnet50(weights="IMAGENET1K_V1")
-    resnet50(pretrained=True)  # deprecated
-    resnet50(True)  # deprecated
```

Comment:
I'm tempted to leave that here until we actually remove support for `pretrained=True`, but no strong opinion

Reply:
Good call, didn't think this through. This made me realize that we failed to schedule a concreate removal date for the legacy model API:

- https://github.com/pytorch/vision/blob/ecbed7bd604a705d4b3709d97817598166b480ca/torchvision/models/_utils.py#L135-L139
- https://github.com/pytorch/vision/blob/ecbed7bd604a705d4b3709d97817598166b480ca/torchvision/models/_utils.py#L208-L211
- https://github.com/pytorch/vision/blob/ecbed7bd604a705d4b3709d97817598166b480ca/torchvision/models/_utils.py#L213-L217

How do we want to go about them? Should we still remove since our documentation stated the removal version? Otherwise what removal version should we put in there?

Reply:
For some context: we had planned to remove them in 0.14 already, and the deprecation message was explicit about that.
When time came to release 0.14, it was decided (after heated discussions and despite concerns from other devs - me in particular) to remove the deprecation deadline in the warning, and to not remove the usage yet, because the `pretrained=True` pattern is still used internally and externally. It was not one of those "disagree and commit" decisions.

I'm personally uncomfortable at the idea that we don't provide a clear removal deadline, and I don't like the fact that we're treating such deprecation on a case-by-case basis. But this isn't something we'll have time to address before 0.15 unfortunately.

Reply:
Let me roll back the doc changes to `pretrained` then and open an issue about this so we don't forget.

Reply:
See #7187.